spec/graphql/functions/find_by_id_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe Functions::FindById do
+
+  describe '.call' do
+    context 'when provided id of exists product' do
+      it 'return product' do
```

Comment:
`when provided id of exists product` -> `when id of existing product is provided`
`returns`